go/vt/vtgate/evalengine/fn_misc.go

```diff
@@ -72,7 +72,7 @@ var _ Expr = (*builtinIsIPV4Compat)(nil)
 var _ Expr = (*builtinIsIPV4Mapped)(nil)
 var _ Expr = (*builtinIsIPV6)(nil)
 
-func (call builtinInetAton) eval(env *ExpressionEnv) (eval, error) {
+func (call *builtinInetAton) eval(env *ExpressionEnv) (eval, error) {
```

Comment:
In #13082 I accidentally added this to non-pointer types, this PR corrects this.